client/web/src/tracking/eventLogger.ts

```diff
@@ -311,9 +321,30 @@ export class EventLogger implements TelemetryService, SharedEventLogger {
             cookies.set(DEVICE_ID_KEY, deviceID, this.cookieSettings)
         }
 
+        let originalReferrer = cookies.get(ORIGINAL_REFERRER_KEY)
+        if (!originalReferrer) {
+            originalReferrer = document.referrer
```

Comment:
I'm open to adding a clause here for it to assign the value of `originalReferrerKey` to be `_mkto_referrer` if it exists, that way we will have parity between the two values. What do you think? @bretthayes 

```suggestion
            originalReferrer = cookies.get(MKTO_ORIGINAL_REFERRER_KEY) || document.referrer
```

Reply:
After learning the intent, I don't think we want to pollute the two differentiated values, right? I think it should be the following:

```suggestion
            originalReferrer = cookies.get(MKTO_ORIGINAL_REFERRER_KEY) || ''
```

And then if it ever happens that we have an empty string, we'll know that something is either wrong with the cookie not being set, or there was never an original referrer and it's direct traffic.

Reply:
Good call-out! I messed up here adding `document.referrer` 😅